terraform/modules/custom-domains/versions.tf

```diff
@@ -0,0 +1,18 @@
+
+terraform {
+  required_version = ">= 1.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4"
+      configuration_aliases = [
+        aws,
```

Comment:
The example [here](https://developer.hashicorp.com/terraform/language/providers/configuration#alias-multiple-provider-configurations) says "The following example declares both the `mycloud` and `mycloud.alternate` provider configuration names" and doesn't include the default provider name. I don't know if it will ignore this or do something unexpected.
```suggestion
```

Reply:
Ahh, good to know. Thanks. 👍